docs/ecosystem/ecosystem.md

```diff
@@ -38,7 +38,8 @@ If you know a useful library that has direct support for ZIO, please consider [s
 - [zio-saga](https://github.com/VladKopanev/zio-saga): Purely functional transaction management with Saga pattern
 - [zio-slf4j](https://github.com/NeQuissimus/zio-slf4j): Referentially transparent logging with slf4j
 - [zio-slick](https://github.com/rleibman/zio-slick): Bridge library between ZIO and Slick Functional Relational Mapping Library
-
+- [zio-kinesis](https://github.com/svroonland/zio-kinesis): ZIO Streams based AWS Kinesis client
```

Comment:
Very minor comment but can we alphabetize? Looks like that was the way the list was supposed to be organized though I realize a couple of items got slightly out of order.